package.json

```diff
@@ -54,6 +54,7 @@
     "slice-ansi": "^6.0.0",
     "source-map-support": "^0.5.21",
     "strip-ansi": "^7.0.1",
+    "ts-dedent": "^2.2.0",
```

Comment:
`dedent` seems unmaintained and had some issues with types so I used `ts-dedent`.